Tizen.web/ImageClassificationPipeline/js/main.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Find the index of maximum value in the given array
+ * @param array the score list of each class
+ * @returns the index of the maximum value
+ */
+function GetMaxIdx(array) {
+    if (array.length === 0){
+        return -1;
+    }
+
+    var max = array[0];
+    var maxIdx = 0;
+
+    for (var i = 0; i< array.length; ++i) {
+        if (array[i] > max) {
+            maxIdx = i;
+            max = array[i];
+        }
+    }
+    return maxIdx;
+}
+
+/**
+ * Get the jpeg image path
+ * @returns image path
+ */
+function GetImgPath() {
+    const MAX_IMG_CNT = 2;
+    if (typeof GetImgPath.count === 'undefined') {
+        GetImgPath.count = 0;
+    }
+    var imgsrc = GetImgPath.count++ % MAX_IMG_CNT;
+    imgsrc = imgsrc.toString().concat(".jpg");
+    return "/res/".concat(imgsrc);
```

Comment:
wonder if it is supported to use template literals

```suggestion
    return `res/${imgsrc}.jpg`
```

Reply:
If this works, it will be alot cleaner for the pipeline description as well :)